.github/workflows/checks.yml

```diff
@@ -141,6 +141,14 @@ jobs:
       - name: Build
         run: |
           poetry run poe build
+      - name: Install Scarb
+      # TODO: improve to download latest release - maybe https://github.com/marketplace/actions/release-downloader
```

Comment:
You left a TODO here without an issue

Reply:
https://github.com/software-mansion/protostar/issues/1814